src/ecalc/libraries/libecalc/common/libecalc/input/yaml_types/components/asset.py

```diff
@@ -0,0 +1,62 @@
+from typing import Dict, Optional
+
+from libecalc.input.yaml_types import YamlBase
+from libecalc.input.yaml_types.variable import Variables
+from pydantic import Field
+
+# Dummy type used for attributes that aren't yet converted to pydantic yaml models
+DummyType = Optional[Dict]
+
+
+class YamlAsset(YamlBase):
```

Comment:
We already got feedback that `asset` is not a representative name, as this varies a lot. We should however find a name that makes sense. But as long as this does not "bleed" over to users, it is ok for the time being

Reply:
It does bleed, and I need to set a name. EcalcModel and Asset are the two that I know we use right now. I decided to keep using Asset while we still have dto.Asset

Reply:
It might bleed, not sure if it does currently